dressForTheWeather/View Controllers/RecommendationViewController.swift

```diff
@@ -46,13 +56,14 @@ extension RecommendationViewController: RecommendationViewDelegate {
         let alertController = UIAlertController(title: "Error", message: description, preferredStyle: .alert)
         let cancelAction = UIAlertAction(title: "OK", style: .default, handler: nil)
         alertController.addAction(cancelAction)
-        present(alertController, animated: true)
+//        present(alertController, animated: true)
```

Comment:
This is that alert that shows all the time, when we really don't need it to. I'll figure out why it's doing that.... later. :)